packages/@sfx/search-data-source-plugin/test/unit/common/search-data-source-plugin.test.ts

```diff
@@ -0,0 +1,50 @@
+import { expect, sinon, stub } from '../../utils';
+import SearchDataSourcePlugin from '../../../src/search-data-source-plugin';
+import { BridgeConfig, Query } from 'groupby-api';
+import * as SearchPackage from 'groupby-api';
+
+describe('SearchPlugin', () => {
+  let searchPlugin: any;
+
+  beforeEach(() => {
+    searchPlugin = new SearchDataSourcePlugin('cvshealth', true);
+  });
+
+  describe('metadata getter', () => {
+    it('should have the name `search-data-source`', () => {
+      expect(searchPlugin.metadata.name).to.equal('search-data-source');
+    });
+
+    it('should not specify any dependencies', () => {
+      expect(searchPlugin.metadata.depends).to.deep.equal([]);
+    });
+  });
+
+  describe('constructor()', () => {
+    it('should create a new instance of Search BrowserBridge with options', () => {
+      const browserBridgeInstance = { a: 'a' };
+      const SearchBrowserBridge = stub(SearchPackage, 'BrowserBridge').returns(browserBridgeInstance);
+      const clientId = 'cvshealth';
+      const https = true;
+      const options: BridgeConfig = {
+        timeout: 2000,
+      };
+
+      searchPlugin = new SearchDataSourcePlugin(clientId, https, options);
+
+      expect(SearchBrowserBridge).to.be.calledWith(clientId, https, options);
+      expect(SearchBrowserBridge.calledWithNew()).to.be.true;
+      expect(searchPlugin.browserBridge).to.equal(browserBridgeInstance);
+    });
+  });
+
+  describe('register()', () => {
+    it('should return the search data source browser bridge instance', () => {
+      const browserBridgeInstance = searchPlugin.browserBridge = { a: 'a' };
+      const registerReturnValue = searchPlugin.register();
+      expect(registerReturnValue).to.equal(browserBridgeInstance);
```

Comment:
To keep consistent, please put newlines between each of these steps (arrange, act, and assert).